### PR TITLE
Extends documentation of MGT components' events

### PR DESCRIPTION
**Pull request by waldekmastykarz**
Extends documentation of MGT components' events.
Closes https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/1220

### CODE REVIEW

File: concepts/toolkit/components/agenda.md [hunk 1]
```diff
@@ -124,9 +124,9 @@ To learn more, see [templates](../customize-components/templates.md).
 
 The following events are fired from the control.
 
-| Event | Description |
-| --- | --- |
-| `eventClick` | The user clicks or taps an event.|
+Event | When is it emitted | Custom data | Cancelable | Bubbles | Works with custom template
+------|-------------------|--------------|:-----------:|:---------:|:---------------------------:|
+`eventClick` | The user clicks or taps an event. | Selected [event](/graph/api/resources/event) | No | No | No
```
Comment (nmetulev) [inline concepts/toolkit/components/agenda.md:129]: I believe the event works with the `event` template, no?

**Review by nmetulev — COMMENTED** [review]
Just WOW, this is amazing

File: concepts/toolkit/components/agenda.md (hunk 1 above)
Comment (waldekmastykarz) [reply concepts/toolkit/components/agenda.md:129]: You're right. My bad

**Comment by waldekmastykarz** [comment]
Just to check: are we ok to merge @nmetulev? 🙂

**Review by beth-panx — APPROVED** [review]

**Comment by beth-panx** [comment]
@Lauragra This PR is ready for your review. :)

**Review by Lauragra — APPROVED** [review]

**Pull request merged**